app/Nova/Metrics/AccessOverrides.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+// phpcs:disable Generic.CodeAnalysis.UnusedFunctionParameter,SlevomatCodingStandard.Functions.UnusedParameter
+
+namespace App\Nova\Metrics;
+
+use App\User;
+use Illuminate\Http\Request;
+use Laravel\Nova\Metrics\Value;
+use Laravel\Nova\Metrics\ValueResult;
+
+class AccessOverrides extends Value
+{
+    /**
+     * Calculate the value of the metric.
+     *
+     * @param \Illuminate\Http\Request  $request
+     *
+     * @return \Laravel\Nova\Metrics\ValueResult
+     */
+    public function calculate(Request $request): ValueResult
+    {
+        return $this->result(User::inactive()->accessActive()->count());
```

Comment:
This is not necessarily true - dues package grace periods will also meet this criteria.